README.md

```diff
@@ -59,60 +72,55 @@ You'll need this package structure to construct a provider package repo:
 ```
 
 
-## Development Standards
+## Part 2: Development Standards
 
-### Building Provider Package
+If you followed the formatting guidelines above, you're now ready to start editing files to include standard package functionality.
 
-Most of what you need is included in `setup.py` and ready to customize. You may use `setup.cfg` as a valid alternative.
+### Python Packaging Scripts
 
-### Provider Readmes
+Your `setup.py` file should contain all of the appropriate metadata and dependencies required to build your package. Use the [sample `setup.py` file](https://github.com/astronomer/airflow-provider-sample/blob/main/setup.py) in this repository as a starting point for your own project.
 
-Readmes should contain top-level documentation about the provider's service, how to build a connection to the service from Airflow, what modules exist within the package, what dependency versions the provider has been tested with, and how the repository maintainers would like folks to contribute.
+If some of the options for building your package are variables or user-defined, you can specify a `setup.cfg` file instead.
 
-#### Managing Dependencies
+### Managing Dependencies
 
-When building providers, these guidelines will help you avoid potential for dependency conflicts.
+When building providers, these guidelines will help you avoid potential for dependency conflicts:
 
-1. It is important that the providers do not include dependencies that conflict with the underlying dependencies for a particular Airflow version. [All of the default dependencies included in the core Airflow project can be found here.](https://github.com/apache/airflow/blob/master/setup.py#L705)
-2. Keep all dependencies upper-bound relaxed; at least allow minor versions, ie. `depx >=2.0.0, <3`. Please include a section in your Readme with the exact set of dependencies that your provider package has been tested with.
+- It is important that the providers do not include dependencies that conflict with the underlying dependencies for a particular Airflow version. All of the default dependencies included in the core Airflow project can be found in the Airflow [setup.py file](https://github.com/apache/airflow/blob/master/setup.py#L705).
```

Comment:
I always avoid using `here` in Hyperlinks, it's best practice to describe where someone will be taken qwhen clicking on something.